### DISCUSSION TITLE
[공지사항] ※2021-01-09 수업 후 공지사항※

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#96] [00_공지사항]
여러분 안녕하세요!

멘토 유혁준입니다.

공지사항 및 과제 말씀드릴게요!

## 과제 공지사항

1. 과제 수정해서 업로드할 때 (2주차 과제는 이렇게 하기!!)
    1. 멘토들이 준 HW 파일 들어가서, 문제 밑에 code 작성란에 작성해서 업로드하기
    2. 멘토들이 코멘트 달아주면 기존 과제파일을 수정해서 업로드하기 (git, github의 장점을 살려봅시다!)
2. 과제 1주차별 review 2번째 달았으니 수정해서 최대한 빨리 다시 pull request 할 것 (과제 문제 끝까지 읽어보시고 풀어주세요!)
3. **issue 확인하셨으면 꼭 댓글로 확인했다고 댓글 달아주세요!** 멘토들이 불안에 떨고 있어요
4. **1주차 issue 정리해서 마크다운으로 정리해서 github에 업로드하기**
5. **멘티들 리뷰 안한 사람들 무조건 하기!**
6. **pandas 과제 출력물 모두 나오게 작성하셔서 github에 업로드하기**
7. **Notion -> 프로젝트 페이지에서 자신이 하고싶은 프로젝트의** **proposal** **을 아래의 양식에 맞춰서 작성하시고** 팀원들을 구하시길 바래요!!
    1. 주제
    2. 왜 이 주제를 해야할까?
    3. 멤버 및 역할 분담
    4. 사용층
    5. 기대 효과
    6. 아키텍쳐
        1. 사용 package or framework
        2. 구성
 8. Dicussion에 아이디어 카테고리에서 논의해보고 싶은 아이디어를 올려보시길 바랄게요!

요즘 늦는 사람이 좀 있는데!ㅠㅠ 시간 지켜주세요!
여러분들에게 알려주고 싶은게 너무 많아서 실제로 못얘기한 부분도 많아요!

그럼 과제하시다 궁금하신 부분이나 모르겠는건 Discussion, Issue 많이 애용해주세요!

**Comment by lsj0822** [comment]
확인했습니다!

**Comment by ghost** [comment]
확인했습니다

**Comment by NA-DK** [comment]
확인했습니다.

**Comment by dlrudgnsdlek** [comment]
확인했습니다.

**Comment by jiwoo4424** [comment]
확인했습니다

**Comment by yoomambo** [comment]
@lsj0822 @NA-DK @dlrudgnsdlek @jiwoo4424 @JaebeomHan 

이번 과제도 저번주처럼 목요일 오후 6시까지 제출!

**Reply by lsj0822** [reply]
확인했습니다
